musicbot/bot.py

```diff
@@ -1781,14 +1781,32 @@ def check(message):
 
 
     async def cmd_disconnect(self, server, message):
+        await self.safe_send_message(
+            message.channel,
+            'Disconnecting from %s...' % server,
+            expire_in=30,
+            also_delete=message if self.config.delete_invoking else None
```

Comment:
Yeah, it works on the disconnect cmd, but not on the restart or shutdown cmds.  I originally left them because I didn't know if they would work without them, but after testing, they have been removed.